handbook/direction/code-graph/code-insights/index.md

```diff
@@ -1,44 +1,218 @@
+
+This means that the Code Insights team is singularly focused on bringing Code Insights into general availability. To reach that goal, Code Insights will be building stability, features, and core product experiences based on the feedback we get from our beta participants.
+
+During the Beta, we are also evaluating pricing and packaging options. It is likely Code Insights will be a standalone feature with its own pricing.
+
+For what we have learned so far and how we are operationalizing those learnings, see [Strategy](#Strategy-and-plans).
+
+### Top customer, support, sales and marketing issues
+
+- The biggest issues customers face, and that we're addressing, are the yet-to-be-built product gaps. See the docs page for [Current Limitations of Code Insights](https://docs.sourcegraph.com/code_insights/explanations/current_limitations_of_code_insights) for an up-to-date list.
+- Internally, the main task for Code Insights is determining the pricing and packaging
+
+### Competitive landscape
+
+While there are many devtool products that use some version of the word "Insights," and a handful of others that do "high level information about code," there are **no known direct compeitors to Code Insights**.
+
+There are no other products that can efficiently and effectively provide the deep historical and realtime analysis that Code Insights can provide, largely because any product that wanted to compete with Sourcegraph Code Insights would first have to catch up to the product moat of Sourcegraph Search (which powers Code Insights), and then also build out a Code Insights-like product.
+
+There may be one-off instances of companies that have glued together in-house solutions to answer some of the same problems, but we have already validated (thus far) every one of those companies would prefer to use Code Insights.
+
+While Code Insights is in Beta, you can find a detailed breakdown of "similar" products in the original [Product Document for Code Insights](https://docs.google.com/document/d/1d34gCpt_rUOMAun8phcjNsFofGaaA_N_8znmgaugdKw/edit#bookmark=id.pobwla63lsa7). At some moment pre-GA launch, the product and marketing team will create more standard competitive resources based on the GA product features and merge them into this page.
+
+## Detailed Strategy and Plans
+
+### Strategy for FY22H2:
+
+The next six months are focused on building a product that can quickly prove out its product-market fit via successful customer sales.
+
+### Goals
+
+#### FY22Q3
+
+- OKR 1: We have qualitative/anecdotal evidence that at least 3 customers have a manager/director/VP who is using code insights
+- OKR 2: 50% of customers on 3.31+ make 10 code insights per customer, and have at least 5 weekly unique viewers of insights ([Looker dashboard](https://sourcegraph.looker.com/dashboards-next/209?Latest+Version=))
```

Comment:
These are really the key results, not the objectives, right?
```suggestion
- KR 1: We have qualitative/anecdotal evidence that at least 3 customers have a manager/director/VP who is using code insights
- KR 2: 50% of customers on 3.31+ make 10 code insights per customer, and have at least 5 weekly unique viewers of insights ([Looker dashboard](https://sourcegraph.looker.com/dashboards-next/209?Latest+Version=))
```

Reply:
Sometimes I think it can be read as "Objective's key result" but sure, also works :)